Sources/SwiftSyntaxBuilder/gyb_generated/Tokens.swift

```diff
@@ -19,178 +19,340 @@ enum Tokens {
+  static let `poundElseif` = SyntaxFactory.makePoundElseifKeyword()
+    .withLeadingTrivia(.spaces(1))
   /// The `#endif` keyword
-  static let `poundEndif` = SyntaxFactory.makePoundEndifKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundEndif` = SyntaxFactory.makePoundEndifKeyword()
+    .withLeadingTrivia(.spaces(1))
   /// The `#available` keyword
-  static let `poundAvailable` = SyntaxFactory.makePoundAvailableKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundAvailable` = SyntaxFactory.makePoundAvailableKeyword()
+    .withLeadingTrivia(.spaces(1))
   /// The `#fileLiteral` keyword
-  static let `poundFileLiteral` = SyntaxFactory.makePoundFileLiteralKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundFileLiteral` = SyntaxFactory.makePoundFileLiteralKeyword()
+    .withLeadingTrivia(.spaces(1))
   /// The `#imageLiteral` keyword
-  static let `poundImageLiteral` = SyntaxFactory.makePoundImageLiteralKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundImageLiteral` = SyntaxFactory.makePoundImageLiteralKeyword()
+    .withLeadingTrivia(.spaces(1))
   /// The `#colorLiteral` keyword
-  static let `poundColorLiteral` = SyntaxFactory.makePoundColorLiteralKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundColorLiteral` = SyntaxFactory.makePoundColorLiteralKeyword()
+    .withLeadingTrivia(.spaces(1))
 
   // MARK: Punctuations and Signs
 
-  /// `":"`
-  static let colon = SyntaxFactory.makeColonToken().withTrailingTrivia(.spaces(1))
+  /// The `(` keyword
+  static let `leftParen` = SyntaxFactory.makeLeftParenKeyword()
+  /// The `)` keyword
+  static let `rightParen` = SyntaxFactory.makeRightParenKeyword()
+  /// The `{` keyword
+  static let `leftBrace` = SyntaxFactory.makeLeftBraceKeyword()
+    .withLeadingTrivia(.spaces(1))
+  /// The `}` keyword
+  static let `rightBrace` = SyntaxFactory.makeRightBraceKeyword()
+    .withLeadingTrivia(.spaces(1))
+  /// The `[` keyword
+  static let `leftSquareBracket` = SyntaxFactory.makeLeftSquareBracketKeyword()
+  /// The `]` keyword
+  static let `rightSquareBracket` = SyntaxFactory.makeRightSquareBracketKeyword()
+  /// The `<` keyword
+  static let `leftAngle` = SyntaxFactory.makeLeftAngleKeyword()
+    .withLeadingTrivia(.spaces(1))
+    .withTrailingTrivia(.spaces(1))
+  /// The `>` keyword
+  static let `rightAngle` = SyntaxFactory.makeRightAngleKeyword()
+    .withLeadingTrivia(.spaces(1))
+    .withTrailingTrivia(.spaces(1))
+  /// The `.` keyword
+  static let `period` = SyntaxFactory.makePeriodKeyword()
+  /// The `.` keyword
+  static let `prefixPeriod` = SyntaxFactory.makePrefixPeriodKeyword()
+  /// The `,` keyword
+  static let `comma` = SyntaxFactory.makeCommaKeyword()
+    .withLeadingTrivia(.spaces(1))
+  /// The `...` keyword
+  static let `ellipsis` = SyntaxFactory.makeEllipsisKeyword()
+  /// The `:` keyword
+  static let `colon` = SyntaxFactory.makeColonKeyword()
```

Comment:
Should we also change `SyntaxFactory.makeColonKeyword()` -> `SyntaxFactory.makeColonToken()` @ahoppen?

Reply:
I guess so because it doesn’t compile otherwise or does it?

I suggest you look at [SyntaxFactory.swift.gyb](https://github.com/apple/swift-syntax/blob/main/Sources/SwiftSyntax/SyntaxFactory.swift.gyb#L104) to figure out how the call to `SyntaxFactory` should look like so it compiles. 

I also just noticed that there are tokens that don't have a fixed text (for which `token.text` is not set). For those, we might need to generate functions instead of constants. E.g.

```swift
static func identifier(_ text: String) {
  return SyntaxFactory.makeIdentifier(text)
}
```

Reply:
Ah yes! Good one. 
In that way we can also keep the `keyword` in the docs. 

Pushed new updated code